client/components/GetDiagram.js

```diff
@@ -1,19 +1,19 @@
-import React, { useState, useEffect } from 'react';
-import { getDiagram } from '../pages/api/api';
+import React, { useState, useEffect } from "react";
+import { getDiagram } from "../pages/api/api";
 
 export const GetDiagram = ({ TikzCode }) => {
```

Comment:
Minor nit: The React core team prefers people to make props camelCase

https://legacy.reactjs.org/docs/dom-elements.html#:~:text=In%20React%2C%20all%20DOM%20properties,attributes%2C%20which%20should%20be%20lowercased.